doc/Command-Reference.md

```diff
@@ -3857,12 +3857,19 @@ This command is used for administratively bringing up the Physical interface or
   admin@sonic:~$ sudo config interface startup Ethernet8,Ethernet16-20,Ethernet32
   ```
 
+**config interface <interface_name> speed (Versions >= 202006)**
+
+Dynamic breakout feature is supported in SONiC from 202006 version.
+User can configure any speed specified under "breakout_modes" keys under the parent interface in the platform-specific port configuration file (i.e. platform.json). Remember, Speed specified in the current breakout mode are the only available speed for the interface at that point of time.
+
+For details please refer [DPB HLD DOC](https://github.com/Azure/SONiC/blob/master/doc/dynamic-port-breakout/sonic-dynamic-port-breakout-HLD.md#cli-design) to know more about this command.
+
 **config interface speed <interface_name> (Versions >= 201904)**
 
 **config interface <interface_name> speed (Versions <= 201811)**
 
 This command is used to configure the speed for the Physical interface. Use the value 40000 for setting it to 40G and 100000 for 100G. Users need to know the device to configure it properly.
-Dynamic breakout feature is yet to be supported in SONiC and hence uses cannot configure any values other than 40G and 100G.
+Dynamic breakout feature is not supported in SONiC yet.
```

Comment:
Is this true? I know NVIDIA will not be supporting until 202111 but to my knowledge the infrastructure should be supported.

Reply:
removed the line to avoid the conflicts for users. Dynamic breakout feature is supported from 202006 as per SONiC release notes. [202006_Release_Notes](https://github.com/Azure/SONiC/blob/master/doc/SONiC_202006_Release_Notes.md#dynamic-port-breakout)